lib/internal/streams/operators.js

```diff
@@ -174,11 +177,25 @@ async function * filter(fn, options) {
   yield* this.map(filterFn, options);
 }
 
+async function toArray(options) {
```

Comment:
```suggestion
/**
 * @param {{ signal?: AbortSignal }} [options]
 * @returns {Array<unknown> | Buffer}
 */
async function toArray(options = undefined) {
```

Reply:
```suggestion
/**
 * @param {{ signal?: AbortSignal }} [options]
 * @returns {Buffer | any[]}
 */
async function toArray(options = undefined) {
```

Reply:
We don't have JSDoc comments for any of the other functions in this file, should we maybe do a PR adding it to all of them at once?

Reply:
@aduh95 the `.length` is different in all the other methods (that take a signal), we've opened an issue to ensure forward-compatibility to ensure that.

Given the primary proposal contributors from what I can tell @devsnek and @ExE-Boss are node contributors I will happily accept guidance with regards to how strict we should be about it :)